### DISCUSSION TITLE
Episode 146: 🇸🇮 SRT23 - Algorithms, BQN's Superpowers & More!

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#37] [Episode]
Episode Link: https://adspthepodcast.com/2023/09/08/Episode-146.html

Feel free to write comments or ask questions 🙂

**Comment by oantolin** [comment]
Towards the end of the episode Connor mentioned the under operator and said "structural under" but proceeded to give examples only of the other kind of under, "mathematical under". Here's a [good blog post](https://www.dyalog.com/blog/2023/01/structural-vs-mathematical-under/) about the difference. Note that J also has mathematical under, but lacks structural under, and BQN has both kinds.
